src/main/java/org/hyperskill/hstest/stage/SpringTest.java

```diff
@@ -154,12 +156,19 @@ public void tearDown() {
     }
 
     public static void startSpring() throws Exception {
+        boolean isKotlin = false;
         if (!springRunning) {
             String annotationPath = "org.springframework.boot.autoconfigure.SpringBootApplication";
-            List<Class<?>> suitableClasses = ReflectionUtils.getClassesAnnotatedWith(annotationPath)
-                    .stream()
-                    .filter(ReflectionUtils::hasMainMethod)
-                    .collect(Collectors.toList());;
+            List<Class<?>> suitableClasses = ReflectionUtils.getClassesAnnotatedWith(annotationPath);
+            List<String> allNameOfClasses = ReflectionUtils.getAllClassesFromPackage("")
+                    .stream().map(Class::getCanonicalName)
+                    .collect(Collectors.toList());
+            for (String s : allNameOfClasses) {
+                if (s.contains("ApplicationKt")) {
```

Comment:
I would rather use the condition `s.endsWith("Kt")` instead of `s.contains("ApplicationKt")` because users doesn't obligated to name their classes/files `Application`. They may choose other names instead, and in that case such condition will fail.

Reply:
Also, it's better to use more meaningful variable names such as `className` or just `name` at the very least. Using just `s` is not that good.

Reply:
Oops, you are right, `s `it's not better variant.